Do not include <malloc.h> on Darwin which does not have that nonstandard
header. malloc is defined in <stdlib.h> which is already included.
--- commondefs.h.orig	2012-09-22 21:04:41.000000000 -0500
+++ commondefs.h	2021-03-19 02:15:04.000000000 -0500
@@ -34,7 +34,7 @@
 /* Shriram: end */
 
 /* cgh */
-#if !defined(vms)
+#if !defined(vms) && !defined(__APPLE__)
 #include <malloc.h>
 #endif
 
--- unknown.h.orig	2006-09-06 14:26:14.000000000 -0500
+++ unknown.h	2021-03-19 02:15:25.000000000 -0500
@@ -17,7 +17,7 @@
 #include <string.h>
 
 /* cgh */
-#if !defined(vms)
+#if !defined(vms) && !defined(__APPLE__)
 #include <malloc.h>
 #endif
 
